pkg/operator/certrotationcontroller/certrotationcontroller.go

```diff
@@ -21,6 +21,9 @@ import (
 	"github.com/openshift/library-go/pkg/operator/v1helpers"
 )
 
+// rotationDay is one day. Set this to short values for testing, e.g. 5 minutes.
+const rotationDay = 24 * time.Hour
```

Comment:
can we make this var and get the value via function? so in case we want to make this much shorter (like 1 * minute) there is a way to do it (either by a hidden flag or env var.)

Reply:
The constants as defined in the installer codebase was clearer in intent: https://github.com/openshift/installer/blob/0be3dfc7f6c6444d17d2edb86e2c52e9d008f9d7/pkg/asset/tls/tls.go#L21-L32

```
const (
	keySize = 2048

	// ValidityOneDay sets the validity of a cert to 24 hours.
	ValidityOneDay = time.Hour * 24

	// ValidityOneYear sets the validity of a cert to 1 year.
	ValidityOneYear = ValidityOneDay * 365

	// ValidityTenYears sets the validity of a cert to 10 years.
	ValidityTenYears = ValidityOneYear * 10
)
```

Reply:
This is not normal users will tweak and we will discourage anyone from tweaking this except us debugging issues related with rotation where we need to decrease the interval to like 5 minutes.

I think as @sttts mentioned, we just need an unit that we can override, which is why I suggested this to not be a const but a func that can read the unit (minute, hour...) from either an env var or some hidden flag of operator binary.

Reply:
It was not meant to be configurable code, but cleaner code. Get the value from a function that handles the Refresh and Validity values creates consistency: https://github.com/openshift/cluster-kube-apiserver-operator/pull/338#discussion_r266485530

Seems it already got merged